LEAF_Request_Portal/templates/subindicators.tpl

```diff
@@ -116,7 +116,13 @@
                         return !valid;
                     },
                     setRequiredError: function() {
+                        $([document.documentElement, document.body]).animate({
```

Comment:
With all of these animations happening on the page at once, it should obey on the first animate, stop, and clear the queue of all future animations until the "Next Question" button is clicked.  If you don't, a page with multiple animates will scroll, pause, scroll, pause all the way down the page to the last required error.  

Look into the use of stop(), finish() or clearQueue() jquery functions to find out the combination you need.

Reply:
👍 Changed